test/src/databases/all/expr.spec.ts

```diff
@@ -228,8 +228,11 @@ describe.each(runtimes.runtimeList)('%s', (databaseName, runtime) => {
   });
 
   // TODO not sure why this test needs to be skipped on postgres, feels like an oversight
-  testIf(databaseName !== 'postgres')('model: dates named', async () => {
-    await expect(`
+  // NOTE: unless underlying type is stored as a timestamp snowflake does not support extraction
```

Comment:
@mtoy-googly-moogly probably need a longer-term solution here...

Reply:
We absolutely need to re-think how we write and run tests across dialects. We grew things organically with Looker and it turned into a swamp. At some point we need to drain the Malloy swamp while it is still small, and make something that will be maintainable going forwards. It is a big project.